src/test/java/Homework19.java

```diff
@@ -0,0 +1,26 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+
+    @Test
+    @Parameters ({"BaseURL", "HomeURL"})
+    public static void deletePlaylist () throws InterruptedException {
+        launchBrowser("https://bbb.testpro.io/");
+        login("prytulan@gmail.com", "28268286nN*");
+        clickSubmit();
+        getHome("https://bbb.testpro.io/#!/home");
```

Comment:
It's not making sense of the @parameters annotation if you are passing a hardcoded string into a function. If you want to use a parameter you can do it this way:
![A31-QA-Automation-Framework – Homework19 java  A31-QA-Automation-Framework test](https://user-images.githubusercontent.com/49252454/211636207-ec323bc8-b0c0-45ef-a350-31451ad40f78.jpg)